jenkins/flow/master/edx-platform-bok-choy-master.groovy

```diff
@@ -35,39 +36,39 @@ try{
   guard{
     parallel(
         {
-          bok_choy_1 = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "bok-choy", ENV_VARS: params["ENV_VARS"], PARENT_BUILD: "master #" + build.number)
+          bok_choy_1 = build(subsetJob, sha1: sha1, SHARD: "1", TEST_SUITE: "bok-choy", ENV_VARS: params["ENV_VARS"], PARENT_BUILD: "master #" + build.number, WORKER_LABEL: workerLabel)
           toolbox.slurpArtifacts(bok_choy_1)
```

Comment:
Not necessarily part of this pr, but could we do this w/ a loop, rather than copy/pasting for each shard?